src/components/ProjectCard/ProjectCard.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import "./ProjectCard.css";
+
+import { Link } from "react-router-dom";
+
+export default function ProjectCard(props) {
+    const { year, title, img, technologies, bgColor } = props;
+    return (
+        <div
+            style={{ backgroundColor: bgColor }}
+            className="project-card-container"
+        >
+            <div className="project-card-info">
+                <p>{year}</p>
+                <strong>
+                    <h2>{title}</h2>
+                </strong>
+                <p>Full Stack Development</p>
+                {technologies.map((tec) => {
+                    return (
+                        <img
+                            className="project-card-icons"
+                            key={tec}
+                            src={tec}
+                            alt={tec}
```

Comment:
this alt will be something like `"/icons/react.png"` this is not useful to a screen reader ( accessibility ) .. maybe you can do something like : 
`const alt = tec.split("/")[tec.split("/").length - 1].split(".")[0];`
and use it here

Reply:
done.
i don't think i totally understood how you did it actually! will come back to it later :)